modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -24,82 +24,80 @@ This page describes the options on the *Deploy Application* panel that is identi
+. In *Package Explorer*, right-click the project node, then select *Anypoint Platform* > *Deploy to CloudHub*.
 +
-image::deploy-to-cloudhub.png[deploy+to+cloudhub]
+// image::deploy-to-cloudhub.png[deploy+to+cloudhub]
 +
-. If this is your first time deploying in this way, a popup menu asks you to provide your login credentials for the Anypoint Platform. Studio stores your credentials and uses them automatically the next time you deploy to CloudHub.
+. If this is your first time deploying from Studio, a popup dialog box asks you to provide your login credentials for Anypoint Platform. Studio stores your credentials and uses them automatically the next time you deploy to CloudHub.
 
 +
 [TIP]
 You can manage these credentials through the Studio *Preferences* menu, in *Anypoint Studio* > *Authentication*.
 
-. After you sign in, the Deploy Application menu opens.
+. After you sign in, the *Deploy Application* dialog window opens.
 +
 image::deployappfirstscreen.png[DeployAppFirstScreen]
 
+[NOTE]
+--
+You cannot deploy an application from Studio to the CloudHub *Design* environment. The *Design* environment is specific to applications created in Anypoint Platform Design Center.
+--
+
 [TIP]
 For an example of deploying to CloudHub directly from Anypoint Studio, see xref:general:getting-started:deploy-to-cloudhub.adoc[Deploy to CloudHub].
 
 == Creating an Application Name
 
-Every application requires a unique application domain name. The application domain identifies your application in the Runtime Manager and also provides you with a public URL, `+http://myapplicationdomain.cloudhub.io+`, for accessing the application if it exposes any inbound endpoints.
+The application name becomes the `cloudhub.io` domain name that you use to access your application, therefore, application names must be unique to avoid domain conflicts. The application domain identifies your application in Runtime Manager and also provides you with a public URL for accessing the application if it exposes any inbound endpoints. The format of the URL is `+http://myapplicationdomain.cloudhub.io+`. For example, an application named `abcde` is accessible at `+http://abcde.cloudhub.io+`.
 
-In the *Deploy Application* dialog, enter a unique application name. The Application Name also becomes the `cloudhub.io` domain name that you use to access your application; for example the Application Name, `abcde` is accessible at `+http://abcde.cloudhub.io+`. Valid names contain alphanumeric characters and dashes, and contain at most 42 characters. If the name is valid, the Runtime Manager alerts you whether it is available or already reserved by another user.
+In the *Deploy Application* dialog, enter a unique application name. Valid names have a maximum of 42 characters, which can contain alphanumeric characters and dashes. If the name is valid, Runtime Manager alerts you as to whether it is available or already reserved by another user.
```

Comment:
"which" seems to refer to 42 chars, not "valid names". so I'd restructure or just say "and" instead of "which"?

Reply:
"as to whether" seems a bit complicated? Maybe "alerts you if the name is available...." ("it" isn't super clear antecedent-wise).